components/Dialogs/README.md

```diff
@@ -130,29 +134,37 @@ dialogTransitionController = MDCDialogTransitionController()
 myDialogViewController.modalPresentationStyle = .custom
 myDialogViewController.transitioningDelegate = dialogTransitionController
 
+// Material theming of presentation controller (see full syntax below)
+myDialogViewController.mdc_dialogPresentationController.applyTheme(withScheme: scheme)
+
 present(myDialogViewController, animated: true, completion:...)
 ```
 
 #### Objective-C
 
 ```objc
-// self is the presenting view controller and which has the following property
+// self is the presenting view controller which has the following property
 // defined to keep a reference to the transition controller.
 @property(nonatomic) MDCDialogTransitionController *dialogTransitionController;
 
-// To present the dialog myDialogViewController
+// Prepare to present the dialog myDialogViewController
 self.dialogTransitionController = [[MDCDialogTransitionController alloc] init];
 myDialogViewController.modalPresentationStyle = UIModalPresentationCustom;
 myDialogViewController.transitioningDelegate = self.dialogTransitionController;
+
+// Material theming of presentation controller (see full syntax below)
+[myDialogViewController.mdc_dialogPresentationController applyThemeWithScheme: scheme];
+
 [self presentViewController:myDialogViewController animated:YES completion:...];
 
 ```
-<!--</div>-->
```

Comment:
Was this removed by accident? It seems like it will fail to terminate the `material-code-render` div.

Reply:
+1: please revert this line change.